nautobot/extras/tests/test_api.py

```diff
@@ -1716,7 +1718,7 @@ def test_run_job_response_job_result(self, mock_get_worker_count):
         response = self.client.post(url, {"data": job_data}, format="json", **self.header)
         self.assertHttpStatus(response, self.run_success_response_status)
 
-        job_result = JobResult.objects.get(name=self.job_model.name)
+        job_result = JobResult.objects.latest()
```

Comment:
Why this change? Should we at least include a `filter(name=self.job_model.name)` before the `.latest()`?